app/scripts/controllers/createPersistentVolumeClaim.js

```diff
@@ -78,7 +78,7 @@ angular.module('openshiftConsole')
           var unit =  $scope.claim.unit || "Mi";
           pvc.spec.resources.requests.storage = $scope.claim.amount + unit;
           pvc.spec.selector.matchLabels = keyValueEditorUtils.mapEntries( keyValueEditorUtils.compactEntries($scope.claim.selectedLabels) );
-          if ($scope.claim.storageClass !== null) {
+          if ( typeof($scope.claim.storageClass ) !== "undefined" && $scope.claim.storageClass  !== null ) {
```

Comment:
When `$scope.claim.storageClass` is undefined, that `if` will evaluate to false.